src/CurricularAnalytics.jl

```diff
@@ -21,23 +21,29 @@ include("GraphAlgs.jl")
 include("DegreePlanAnalytics.jl")
 include("DegreePlanCreation.jl")
 
-export AA, AAS, AS, BA, BS, Course, CourseCatalog, Curriculum, Degree, DegreePlan, EdgeClass, LearningOutcome, Requisite, System, Term, add_course!, 
-        add_lo_requisite!, add_requisite!, all_paths, basic_metrics, basic_statistics, bin_filling, blocking_factor, centrality, co, compare_curricula, 
-        complexity, course, course_from_id, course_from_vertex, course_id, courses_from_vertices, create_degree_plan, dead_ends, delay_factor, delete_requisite!, 
-        dfs, extraneous_requisites, find_term, gad, homology, is_duplicate, isvalid_curriculum, isvalid_degree_plan, longest_path, longest_paths, merge_curricula, 
-        pre, print_plan, quarter, reach, reach_subgraph, reachable_from, reachable_from_subgraph, reachable_to, reachable_to_subgraph, read_csv, requisite_distance, 
-        requisite_type, semester, similarity, strict_co, topological_sort, total_credits, write_csv, Grade, grade, AbstractRequirement, CourseSet, RequirementSet, 
-        CourseRecord, StudentRecord, TransferArticulation, add_transfer_catalog, add_transfer_course, transfer_equiv
+include("Simulation/Simulation.jl")
+include("Simulation/PassRate.jl")
+include("Simulation/Enrollment.jl")
+include("Simulation/Report.jl")

```

Comment:
Would it make sense to wrap these into one `include("Simulation/Simulation.jl")`? I was envisioning that the Simulation file would define a Simulation module which would bring in the other files

Reply:
Add the includes in Simulation/Simulation.jl 179c41190af4e3708a1c12d1494416deb9d7eb74

For module, since like DegreePlanAnalytics.jl and others don't using module, so may be we shouldn't use module for the simulation for consistency?

Reply:
Good point. For consistency we should do it the way you outlined, but I'm not sure if there are any benefits to putting things into modules...